scripts/refresh-performance-benchmarks.ps1

```
@@ -0,0 +1,498 @@
+    $canonicalDirectory = Split-Path -Parent $canonicalSnapshot
+    [IO.Directory]::CreateDirectory($canonicalDirectory) | Out-Null
+    $stagedSnapshot = Join-Path $canonicalDirectory ".latest.$([Guid]::NewGuid().ToString('N')).tmp"
+    [IO.File]::Copy($candidateSnapshot, $stagedSnapshot, $false)
+    & $validator $stagedSnapshot
+    [IO.File]::Move($stagedSnapshot, $canonicalSnapshot, $true)
+    $stagedSnapshot = $null
+
+    if (-not $SkipVerification) {
+        Write-Host ''
+        Write-Host 'Building and verifying the website...'
+        Invoke-Streaming $npm @('run', 'build:self-host', '--', '--configuration', 'Debug') 'Self-host build'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-Streaming' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)